test/common/provider_trace.c

```diff
@@ -195,7 +195,7 @@ umf_memory_provider_ops_t UMF_TRACE_PROVIDER_OPS = {
     .initialize = traceInitialize,
     .finalize = traceFinalize,
     .alloc = traceAlloc,
-    .free = traceFree,
+    .ext.free = traceFree,
```

Comment:
please sort this list here and in other providers. (ext functions should be together)

Reply:
Why? Maybe `.ext.free` could be still after `.alloc` ?